pkgs/development/libraries/zlog/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  version = "1.2.12";
+  name = "zlog-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/HardySimpson/zlog/archive/${version}.tar.gz";
```

Comment:
Looks like a generated archive, better use fetchzip or fetchFromGitHub to not get hash mismatch in the future.